subprojects/core/src/integTest/groovy/org/gradle/configuration/project/LifecycleProjectEvaluatorIntegrationTest.groovy

```diff
@@ -132,5 +151,13 @@ class LifecycleProjectEvaluatorIntegrationTest extends AbstractIntegrationSpec {
             children*.displayName == ["Apply script after.gradle to project ':foo'"]
             parentId == configOp.id
         }
+
+        def configureIncludedBuild = operations.only(ConfigureProjectBuildOperationType, {it.details.buildPath== ':included-build'})
+
+        with(operations.only(NotifyProjectAfterEvaluatedBuildOperationType, {it.details.buildPath == ':included-build'})) {
+            displayName == 'Notify afterEvaluate listeners of :included-build'
+            // parent is not the plugin application operation, as we fire the build op when hooks are executed, not evaluated.
```

Comment:
This comment is a little confusing because we often overload the terms "execute" and "evaluate". I'd replace "evaluated" here with "registered" or some similar nomenclature.